docs/es/docs/deployment/docker.md

```diff
@@ -1,51 +1,51 @@
-# Deploy with Docker
+# Deploy con Docker
 
-In this section you'll see instructions and links to guides to know how to:
+En esta sección verás instrucciones y enlaces a guías para saber como:
 
-* Make your **FastAPI** application a Docker image/container with maximum performance. In about **5 min**.
-* (Optionally) understand what you, as a developer, need to know about HTTPS.
-* Set up a Docker Swarm mode cluster with automatic HTTPS, even on a simple $5 USD/month server. In about **20 min**.
-* Generate and deploy a full **FastAPI** application, using your Docker Swarm cluster, with HTTPS, etc. In about **10 min**.
+* Hacer de tu app con **FastAPI**, una imagen/contenedor de Docker con un máximo rendimiento. En cerca de **5 min**.
+* (Opcionalmente) entender que, como desarrollador, necesitas conocer sobre HTTPS.
+* Configurar un cluster de Docker Swarm mode con HTTPS automatico, incluso en un simple servidor por $5 USD/mes. En cerca de **20 min**.
+* Generar y desplegar en toda su totalidad una aplicación con **FastAPI**, usando tu cluster de Docker Swarm, con HTTPS, etc. En cerca de **10 min**.
 
-You can use <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> for deployment. It has several advantages like security, replicability, development simplicity, etc.
+Usar <a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> para el deployment, tiene algunas ventajas como seguridad, replicabilidad, simplicdad en el desarrollo, etc.
```

Comment:
* change `Usar` for `Puedes usar`
* `several` translation is `varias` `distintas`
* change `simplicdad` for `simplicidad`